### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Splend1ed**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.gitignore
+README.md
+result.png
```
Comment (IvanRamyk) [inline .dockerignore:3]: what about venv?

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,18 @@
+import requests
+import os
+
+
 def get_weather():
-    # write your code here
-    pass
+    key = os.environ["API_KEY"]
```
Comment (IvanRamyk) [inline app/main.py:6]: could be a global constant

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+    city = "Paris"
```
Comment (IvanRamyk) [inline app/main.py:7]: could be a global constant

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+    url = f"https://api.weatherapi.com/v1/current.json?key={key}&q={city}"
```
Comment (IvanRamyk) [inline app/main.py:8]: URL could be a global constant (or at least part without arguments)

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+requests==2.28.1
```
Comment (IvanRamyk) [inline requirements.txt:1]: use `pip freeze > requirements.txt` to save versions for all dependencies

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
You haven't modify readme :(

File: requirements.txt (hunk 5 above)
Comment (Splend1ed) [reply requirements.txt:1]: But it works, if I add more requirements, the image will be bigger, right?

**Review by IvanRamyk — COMMENTED** [review]
Good job!

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Pull request closed without merging**